batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+/**
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/19/16
+ */
+public abstract class CustomHttpServlet extends HttpServlet{
+    private static Logger logger = Logger.getLogger(CustomHttpServlet.class.getName());
+    private static final String ERROR_MESSAGE = "errorMessage";
+
+    public void showPageNotFound(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        req.getServletContext().getRequestDispatcher("/WEB-INF/views/error.jsp").forward(req, resp);
+    }
+
+    public void showServerError(HttpServletRequest req, HttpServletResponse resp, Throwable e) throws ServletException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: javax.servlet.ServletException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)